CHANGELOG.asciidoc

```diff
@@ -3,6 +3,71 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.2.0]]
+=== Beats version 8.2.0
+https://github.com/elastic/beats/compare/v8.1.3...v8.2.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Fix mapping of parent process information provided by add_process_metadata. {issue}29874[29874] {pull}30727[30727]
```

Comment:
@andrewkroh @andresrc @efd6 @epixa Is there any way users can work around this breaking change? Was there any further discussion around making this breaking change? https://github.com/elastic/beats/pull/30727 I like that we aligned to ECS but it should also not mean we just break users in a minor release.

Reply:
If a user wanted the old output they could change their config to `add_process_metadata: {target: process.parent.parent, ...}` (two `.parent` keys since it only filters one). It really seems like this was a bug in the first place that it produced the output that it did, because there was no mapping for the fields and it was not consistent with ECS.

Reply:
Got it, so we consider this a bug where the bugfix is a breaking change. 👍